.github/ISSUE_TEMPLATE/new-concept-exercise.md

```diff
@@ -0,0 +1,205 @@
+
+  * You can refer to one or more of the resources linked above, or analogous resources from a trusted source.  We prefer using links within the  [Python Docs](https://docs.python.org/3/) as the primary go-to, but other resources listed above are also good.
+
+    
+
+* ### `links.json`
+
+  *  The same resources can be used for the [ `concepts/links.json`](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md#file-linksjson)  file, if it doesn't already exist.
+  *  If there are particularly good/interesting informations sources for this concept that extend or supplement the concept exercise material, please add them to this document.
+
+
+
+## Concept Description
+
+[Link to the concept `about.md` file in the Concept section below.  If the file still needs to be written, please link to the **GitHub issue** filed for the concept `about.md` file.]
+
+
+
+Please see the following for more details on these files:  [concepts](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md#file-linksjson)
+
+* ### Concept `about.md`
+
+  [<concept-SLUG>]() and/or [<GitHub issue link>]()
+
+* ### Exercise `introduction.md`
+
+  For more information, see [Exercise `introduction.md`](https://github.com/exercism/docs/blob/main/anatomy/tracks/concept-exercises.md#file-docsintroductionmd)
+
+​           This should summarize  the above document, with  enough information for the student to complete the tasks in the concept exercise.
+
+* ### Concept `introduction.md`
+
+  For more information, see [Concept `introduction.md`](https://github.com/exercism/docs/blob/main/anatomy/tracks/concepts.md#file-introductionmd)
+
+​         This should also be a summary of the above document, and will provide a brief introduction of the concept for a student who has **not yet** completed the concept exercise.
+
+
+
+## Representer
+
+No changes required.
+
+
+
+## Analyzer
+
+No changes required.
+
+
+
+## Exercise UUID
+
+[please generate a UUID for this exercise for `config.json`]
+
+one of these tools can be used, or you can use your favorite method of generating a **V4 UUID**
+
+[exercism configlet](https://github.com/exercism/configlet)
+
+[uuidgenerator.net](https://www.uuidgenerator.net/version4)
+
+
+
+**Exercise UUID** : `<exercise-UUID>`
+
+
+
+## Implementation Notes
+
+In General, tests should be written using `unittest.TestCase` and the test file should be named `<EXERCISE-NAME>_test.py`.
+
+While we do use PyTest as our test runner and for some implantation tests, please check with a maintainer before using  a PyTest test method, fixture,  or feature.
+
+Code in the `.meta/examplar.py` file should **only use syntax & concepts introduced in this exercise or one of its prerequisite exercises.** 
+Please **do not use** comprehensions, generator expressions, or other syntax not previously covered.  Please also follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.
```

Comment:
```suggestion
Code in the `.meta/examplar.py` file should **only use syntax & concepts introduced in this exercise or one of its prerequisite exercises.** 
Please **do not use** comprehensions, generator expressions, or other syntax not previously covered.  Please also follow [PEP8](https://www.python.org/dev/peps/pep-0008/) guidelines.

In General, tests should be written using `unittest.TestCase` and the test file should be named `<EXERCISE-NAME>_test.py`.

While we do use [PyTest](https://docs.pytest.org/en/stable/) as our test runner and for some implementation tests, please check with a maintainer before using  a PyTest test method, fixture,  or feature.

Our markdown and JSON files are checked against [prettier](https://prettier.io/) .  We recommend [setting prettier up locally](https://prettier.io/docs/en/install.html) and running it prior to submitting your PR  to avoid any CI errors.  
```